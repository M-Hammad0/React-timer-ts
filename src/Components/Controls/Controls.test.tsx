import React from 'react'
import '@testing-library/jest-dom'
import Control from './Controls'
import { render, cleanup } from '@testing-library/react';
import setTimeInSeconds from '../Main/Main'

afterEach(cleanup)

it('render control component',() => {
      const {asFragment} = render(<Control time={setTimeInSeconds}/>)
      expect(asFragment).toMatchSnapshot();
})

it('render control buttons', () => {
      const {getByTestId} = render(<Control time={setTimeInSeconds} />)
      expect(getByTestId('btn-play')).toHaveTextContent('Play')
      expect(getByTestId('btn-stop')).toHaveTextContent('Stop')
      expect(getByTestId('btn-reset')).toHaveTextContent('Reset')
})
