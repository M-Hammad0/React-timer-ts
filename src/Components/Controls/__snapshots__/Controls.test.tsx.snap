// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render control component 1`] = `[Function]`;
